### DISCUSSION TITLE
Custom inline styles

### DISCUSSION THREAD

**Discussion by ntgussoni** [markdoc/markdoc#148] [Q&A] [answered]
One thing that you can do in asciidoc is add custom classes or styles to a piece of text in a sentence.
See: https://docs.asciidoctor.org/asciidoc/latest/text/custom-inline-styles/

``` This lets #this words#{% .important %} to be wrapped in a span a different class (or other attributes).```

Is this something you'd be interested in supporting (if it's even possible)?

**Comment by ntgussoni** [comment, marked as answer]
Thanks for the time btw, I've been asking questions like crazy. But I'm very close to replacing asciidoc / asciidoctor completely

**Reply by mfix-stripe** [reply]
You can wrap inline text with your own custom tags, and then do whatever you would like to do with them:

```md
Wrapping {% custom-tag %}this work{% /custom-tag %} with an inline tag
```

**Reply by mfix-stripe** [reply]
You can also use `**` and `_` just like you would do with regular Markdoc, and define what those nodes render using custom nodes.

**Reply by mfix-stripe** [reply]
Finally, you can use class annotations to apply a custom class to an _entire_ node: https://markdoc.dev/docs/syntax#annotations
